### PR TITLE
Add support for vmware_guest:connected.

### PR DESCRIPTION
**Pull request by kdienes**
##### SUMMARY
This adds support for setting the 'connected' status of a NIC from vmware_guest.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
vmware_guest

##### ANSIBLE VERSION
latest (5242914ae5ac15ea5d5e7ba3607bb6c8eb9c72a6)

##### ADDITIONAL INFORMATION
Lightly tested.  I am happy to revise if I've missed anything or if there is a place I should be adding test cases or additional documentation.

### CODE REVIEW

File: lib/ansible/modules/cloud/vmware/vmware_guest.py [hunk 1]
```diff
@@ -215,6 +215,7 @@
     - ' - C(dns_servers) (string): DNS servers for this network interface (Windows).'
     - ' - C(domain) (string): Domain name for this network interface (Windows).'
     - ' - C(wake_on_lan) (bool): Indicates if wake-on-LAN is enabled on this virtual network adapter. version_added: 2.5'
+    - ' - C(connected) (bool): Sets if virtual network adapter is connected. version_added: 2.5'
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/vmware/vmware_guest.py]: s/2.5/2.6/

**Comment by Akasurde** [comment]
@kdienes Thanks for contribution. Could you please add integration test for this change ? You can start [here](https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_guest/tasks/network_negative_test.yml).

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
@kdienes Are you still working on this ? needs_info

**Comment by Akasurde** [comment]
@kdienes Any news ?

**Comment by kdienes** [comment]
I'm still interested in completing this patch but haven't had the time to write the tests.  I don't expect to for at least another several weeks.  I'm happy if someone wants to take over the patch, or willing to withdraw the PR and resubmit when I am able if you prefer.

**Comment by mattclay** [comment]
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/102869/87/tests

**Comment by Akasurde** [comment]
Thank you very much for your interest in Ansible. This plugin is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/community.vmware.

Please re-submit this PR in the above repository.

If you have further questions please stop by IRC or the mailing list:

* IRC: #ansible on irc.freenode.net
* mailing list: https://groups.google.com/forum/#!forum/ansible-project

**Pull request closed without merging**